query/network/network_security_group_rdp_access_restricted.sql

```diff
@@ -9,7 +9,7 @@ with network_sg as (
   where
     sg -> 'properties' ->> 'access' = 'Allow'
     and sg -> 'properties' ->> 'direction' = 'Inbound'
-    and sg -> 'properties' ->> 'protocol' ilike 'TCP'
+    and (sg -> 'properties' ->> 'protocol') in ('TCP', 'tcp', '*')
```

Comment:
When we create these resources through TF, to take any case as a parameter, such as "TCP" or "tcp" or even "tCp", hence better we use like in the case of TCP -- suggesting 

`    and ( sg -> 'properties' ->> 'protocol' ilike 'TCP' or sg -> 'properties' ->> 'protocol' = '*')`

`
```
  security_rule {
    access                     = "Allow"
    direction                  = "Inbound"
    name                       = "AllowSSH"
    priority                   = 200
    protocol                   = "TCP"
    source_address_prefix      = "*"
    source_port_range          = "*"
    destination_port_range     = "22-22"
    destination_address_prefix = "*"
  }

```